pkg/prometheus/promcfg_test.go

```diff
@@ -1504,6 +1504,109 @@ alerting:
 	}
 }
 
+func TestAlertmanagerBasicAuth(t *testing.T) {
```

Comment:
can you add a sub-test with version < 2.26? also worth setting the Prometheus version explicitly for the existing test.